guide/tutorial-5-compound-solid-and-physics-attributes-15-minutes.md

```diff
@@ -138,20 +138,20 @@ We recommend values between *8* and *16* for regular use of Webots.
 It's more difficult to explain the behavior of the `ERP` and `CFM` fields.
 These values are directly used by the physics engine to determine how the constraints are solved.
 The default values are well defined for regular use of Webots.
-We recommend to read the `Reference Manual` and the documentation of [ODE](http://ode-wiki.org/wiki/index.php?title=Manual) (physics engine used in Webots) to understand completely their purpose.
+We recommend to read the [Reference Manual](../reference/worldinfo.md) and the documentation of [ODE](http://ode-wiki.org/wiki/index.php?title=Manual) (physics engine used in Webots) to understand completely their purpose.
 
 ### Minor Physics Parameters
 
 There are also other physics parameters which are less useful in a regular use of Webots.
-A complete description of these parameters can be found in the `Reference Manual`.
-Remark simply that the Physics, WorldInfo and ContactProperties nodes contains other fields.
+A complete description of these parameters can be found in the [Reference Manual](../reference/index.md).
```

Comment:
I am not sure about what parameters we are speaking about here... but the link to the reference manual is too vague, we should be more precise, like ../reference/worldinfo.md if we mean WorldInfo parameters. Probably this sentence should be improved to bring more precise and useful information.

Reply:
I thought that this was part of the exercise to let the user find the information by himself and at the same time learn how to browse the reference manual. 
And the precise information and links are given in the next line:
> Remark simply that the [Physics](../reference/physics.md), [WorldInfo](../reference/worldinfo.md) and [ContactProperties](../reference/contactproperties.md) nodes contains other fields.